czarjulius/Banka-React
# #168095015

#### What does this PR do?
A registered user can log in

#### Description of Task to be completed?
- Create login action and types
- Create login reducer and connect to store
- Add functionality to login page

#### How should this be manually tested?
- A registered user should visit `https://banka-beact.herokuapp.com/login` and submit his email and password.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#168095015](https://www.pivotaltracker.com/story/show/168095015)

#### Screenshots (if appropriate)
-N/A

#### Questions:
- N/A
